http_server/main.js

```diff
@@ -35,6 +35,20 @@ app.get('/world-editor', (req, res) => {
 	})
 })
 
+app.get("/canvas-image", (req, res) => {
+  try {
+    const data = Files.inFolder("static/assets")
```

Comment:
remove all error handling. no need for it in this project

Reply:
done